README.md

```diff
@@ -149,41 +160,199 @@ returns something similar to:
+
+```graphql
 query MyQuery {
   allContentfulArticle {
     nodes {
-      avatar {
+      bannerImage {
         src
+        attributes {
+          analyzed_content_warnings
+          analyzed_faces
+          analyzed_tags
+          color_model
+          color_profile
+          content_type
+          custom_fields
+          date_created
+          date_modified
+          dpi_height
+          dpi_width
+          face_count
+          file_size
+          has_frames
+          media_height
+          media_kind
+          media_width
+          origin_path
+          source_id
+          tags
+          uploaded_by_api
+          warning_adult
+          warning_medical
+          warning_racy
+          warning_spoof
+          warning_violence
+        }
       }
     }
   }
 }
+```
+
+returns something similar to:
+
+```json
+{
+  "data": {
+    "allContentfulArticle": {
+      "nodes": [
+        {
+          "bannerImage": {
+            "src": "https://fourbottle.imgix.net/heroes/woman-stirring.jpg",
+            "attributes": {
+              "analyzed_content_warnings": true,
+              "analyzed_faces": true,
+              "analyzed_tags": true,
+              "color_model": "RGB",
+              "color_profile": "c2",
+              "content_type": "image/jpeg",
+              "custom_fields": "\"\"",
+              "date_created": 1625796011,
+              "date_modified": 1642786873,
+              "dpi_height": 72,
+              "dpi_width": 72,
+              "face_count": 1,
+              "file_size": 3411741,
+              "has_frames": false,
+              "media_height": 4000,
+              "media_kind": "IMAGE",
+              "media_width": 6000,
+              "origin_path": "/heroes/woman-stirring.jpg",
+              "source_id": "5f73d9798d5327eb5194d54a",
+              "tags": "{\"Arm\":0.9448454976081848,\"Cup\":0.8950006365776062,\"Drinkware\":0.9177042841911316,\"Glasses\":0.9809675216674805,\"Hand\":0.9596579074859619,\"Joint\":0.9762823581695557,\"Photograph\":0.94278883934021,\"Shoulder\":0.9465579986572266,\"Tableware\":0.949840784072876,\"Vision care\":0.9289617538452148}",
+              "uploaded_by_api": false,
+              "warning_adult": 1,
+              "warning_medical": 1,
+              "warning_racy": 2,
+              "warning_spoof": 1,
+              "warning_violence": 1
+            }
+          }
+        }
+      ]
+    }
+  }
+}
+```
+
+**Note**: Certain fields under `attributes` are returned as strings to provide better [resiliency](https://forums.fauna.com/t/how-to-store-arbitrary-json-object-via-graphql/142/3) when used with GraphQL. Therefore, these fields (`custom_fields`, `tags`, `colors.dominant_colors`) will need to be parsed back into JSON objects after being queried. The example below demonstrates how to do this:
+
+```js
+{
```

Comment:
I think this is meant to be inside a React component, so maybe it could be a litte clearer that this is the case by adding `const Component = () => {` or something as the first line?

Reply:
@sherwinski ➕ 1 to the above ☝🏼